run_e3sm.template.sh

```diff
@@ -398,6 +407,37 @@ case_setup() {
     popd
 }
 
+#-----------------------------------------------------
+custom_pelayout() {
+
+if [[ ${PELAYOUT} == custom-* ]];
+then
+    echo $'\n CUSTOMIZE PROCESSOR CONFIGURATION:'
+
+    # Number of cores per node (machine specific)
+    if [ "${MACHINE}" == "pm-cpu" ]; then
```

Comment:
@mahf708 : could you add more E3SM machines (compy, chrysalis, anvil) here to make it more portable?